src/README.md

```diff
@@ -0,0 +1,15 @@
+# Shadowcat dev
+
+## Dev Setup
+
+1. `npm install` to install dependencies.
+1. `npm start` to start the development server.
+1. Go to `localhost:3333`. The development server will display a blank page.
+1. Open the console. There will be a message/token logged.
+
+## Publish to npm
```

Comment:
I believe you actually fixed the autopublish flow for this repo, right? Should we update this section?